liquidctl/config/load.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+import sys
+
+def get_config_files(file=None, appname='liquidctl', **kwargs):
+    """
+    This will get all the potential file paths on the system for where the config file
+    can be
+
+    This will also handle the -f overrides
+    """
+    files = []
+    if file:
+        files.append(file)
+
+    if sys.platform == 'win32':
+        files.append(os.path.join(os.getenv('APPDATA'), appname, 'config.toml'))
+    elif sys.platform == 'darwin':
+        files.append(os.path.expanduser(os.path.join('~/Library/Application Support', appname, 'config.toml')))
```

Comment:
Needs to consider system-wide settings as well.

Reply:
This is the application configure file location that apple suggests using. I can add an `/etc/liquidctl` dir here as well though

Reply:
There's `/Library` as well:

> `/Library`(macOS only)—Apps that share resources between users store those resources in this version of the Library directory
> —_[The Library Directory Stores App-Specific Files](https://developer.apple.com/library/archive/documentation/FileManagement/Conceptual/FileSystemProgrammingGuide/FileSystemOverview/FileSystemOverview.html#//apple_ref/doc/uid/TP40010672-CH2-SW1)

But perhaps it's better to store this Unix-style config file in a Unix-style path... we may need to consult some Mac OS people on this one.

@CaseySJ, do you have any idea on what we should do here?

Reply:
I am no longer checking these paths on macos, only the XDG and home dir